syntax = "proto3";

package hellosalerno_v2;

service Greeter{
    rpc SayHello(HelloRequest) returns (HelloReply) {}
    rpc SayHelloAgain(HelloRequest) returns (HelloReply) {}
}

message HelloRequest{
    string nome = 1;
}

message HelloReply{
    string msg = 1;
}